cfme/cloud/tenant.py

```diff
@@ -7,27 +7,61 @@
 
 from utils.conf import cfme_data
 from cfme.fixtures import pytest_selenium as sel
-from cfme.web_ui import Region, SplitTable
-
+from cfme.web_ui import Region, SplitTable, toolbar as tb, CheckboxTable, Form, form_buttons
+from utils.appliance import Navigatable
+from utils.appliance.implementations.ui import navigator, CFMENavigateStep, navigate_to
+from navmazing import NavigateToAttribute, NavigateToSibling
+from cfme.cloud.instance import match_page
+from cfme.common import SummaryMixin, Taggable
 
 # Page specific locators
 list_page = Region(
     locators={
         'tenant_table': SplitTable(header_data=('//div[@class="xhdr"]/table/tbody', 1),
-            body_data=('//div[@class="objbox"]/table/tbody', 1))
+                                   body_data=('//div[@class="objbox"]/table/tbody', 1))
     },
     title='Cloud Tenants')
 
-
+create_form = Form([('cloud_provider', ("//button[@id='cloud_tenant_id']"),
+                     ('tenant_name', "//*[@name='name']"))])
+list_tbl = CheckboxTable(table_locator="//div[@id='list_grid']//table")
 details_page = Region(infoblock_type='detail')
 
 
-class Tenant(object):
+class Tenant(Taggable, SummaryMixin, Navigatable):
     def __init__(self, name, description, provider_key):
         """Base class for a Tenant"""
         self.name = name
         self.description = description
         self.provider_key = provider_key
+        Navigatable.__init__(self)
+
+    def create(self, provider_key):
+        """
+            create a new tenant
+        """
```

Comment:
Comments should look like this: https://google.github.io/styleguide/pyguide.html?showone=Comments#Comments